src/app/features/settings/sign-out/sign-out-confirm.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { RouteUrls } from '@shared/route-urls';
+
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { useKeyActions } from '@app/common/hooks/use-key-actions';
+
+import { SignOutDialog } from './sign-out';
+
+interface SignOutProps {
+  isShowing: boolean;
+  onClose(): void;
+}
+
+export function SignOut({ isShowing = false, onClose }: SignOutProps) {
+  const analytics = useAnalytics();
+
+  //  TODO 4370 check if this is enough to track views of previous sign-out route
```

Comment:
@kyranjamie - we used to have a route for this but now it's a standard dialog. 

I am tracking the view of it with `void analytics.track('sign-out')` in a `useEffect`. Is that enough, or did I miss something ?